### PR TITLE
Update Core and NUnit

### PR DESCRIPTION
**Pull request by SeanFeldman**

### CODE REVIEW

File: src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Testing" Version="[7.0.0, 8.0.0)" />
+    <PackageReference Include="NServiceBus.Testing" Version="7.2.0" />
```
Comment (timbussmann) [inline src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj]: this shouldn't be changed as this is a nuget package we're releasing (https://www.nuget.org/packages/NServiceBus.Callbacks.Testing/)

File: src/NServiceBus.Callback.Testing.Tests/NServiceBus.Callback.Testing.Tests.csproj [hunk 2]
```diff
@@ -12,9 +12,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
-    <PackageReference Include="NServiceBus" Version="7.1.1" />
-    <PackageReference Include="NServiceBus.Testing" Version="7.0.0" />
-    <PackageReference Include="NUnit" Version="3.11.0" />
+    <PackageReference Include="NServiceBus.Testing" Version="7.2.0" />
```
Comment (timbussmann) [inline src/NServiceBus.Callback.Testing.Tests/NServiceBus.Callback.Testing.Tests.csproj]: this causes the transitive dependency to resolve to NServiceBus v 7.0.1 instead 7.2 as transitive dependencies resolve to the lowest version. So NServiceBus has to be explicitly referenced here to make sure the tests use 7.2

Comment (timbussmann) [reply src/NServiceBus.Callback.Testing.Tests/NServiceBus.Callback.Testing.Tests.csproj]: fixed

File: src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj (hunk 1 above)
Comment (SeanFeldman) [reply src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj]: Ah, ok. thank you for fixing it.

**Pull request merged**